### ISSUE TITLE
Linux: video stuttering and workaround

### ISSUE THREAD

**Issue by kokoko3k** [reicast/reicast-emulator#1754]
**Please Note: This form is the minimum required information for submitting bugs.**  
**Removing this form may lead to your issue being closed until it is completed.**

Platform: i5-4590 CPU @ 3.30GHz , GTX 1060, Archlinux + proprietary nvidia drivers
Version: r19.07.4, compiled from source

**_Description of the Issue_**
Video not smooth, stuttering

**_Debugging Steps Tested_**
Open a game
Observe stuttering
See that for a 30fps game, fps in the titlebar goes between 29.67 and 30.31 or so.

**What was tried**
  * enable system vsync via nvidia-control-panel or via compositor
  * enable/disable syncronous rendering
  * enable/disable audio frame limiting
  * switch rendering backend between gl41/gles

- [X ] Place an X in the box to confirm you have filled in this form

Issue presents as long as i let reicast to handle synchronization.
Fortunately, there is libstrangle, an LD_PRELOAD hack that lets you limit the framerate and force VSYNC.
Basically, if i start reicast with: > VSYNC=1 strangle 30 reicast or VSYNC=1 strangle 60 reicast (depending on the framerate of the game), then everything becomes butter smooth and sync'ed to vblank, also, in the titlebar fps stay to 30fps (no more oscillations)
But this is a workaround; i hope that the strangle bit may help in debugging the issue here.

Thanks for developing reicast!
